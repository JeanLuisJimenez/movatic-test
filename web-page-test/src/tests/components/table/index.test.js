import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom'
import Table from '../../../components/table'

it('Table', () => {
    const renderedCard = render(<BrowserRouter>
        <Table 
            headers={[
                { name: 'header 1', key: 'header_1' },
                { name: 'header 2', key: 'header_2' }]}
            content={[{
                header_1: 'Row 1 Content',
                header_2: 'Row 1 Content'
            }, {
                header_1: 'Row 2 Content',
                header_2: 'Row 2 Content'
            }]} />
    </BrowserRouter>)

    expect(renderedCard).toMatchSnapshot()
})
