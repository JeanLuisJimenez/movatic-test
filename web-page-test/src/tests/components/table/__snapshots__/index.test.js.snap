// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="table-container"
      >
        <table
          class="table"
        >
          <thead
            class="table-headers"
          >
            <tr>
              <th
                class="table-header"
              >
                #
              </th>
              <th
                class="table-header"
              >
                header 1
              </th>
              <th
                class="table-header"
              >
                header 2
              </th>
            </tr>
          </thead>
          <tbody
            class="table-row"
          >
            <tr>
              <td
                class="table-cell"
              >
                0
              </td>
              <td
                class="table-cell"
              >
                Row 1 Content
              </td>
              <td
                class="table-cell"
              >
                Row 1 Content
              </td>
            </tr>
          </tbody>
          <tbody
            class="table-row"
          >
            <tr>
              <td
                class="table-cell"
              >
                1
              </td>
              <td
                class="table-cell"
              >
                Row 2 Content
              </td>
              <td
                class="table-cell"
              >
                Row 2 Content
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="table-container"
    >
      <table
        class="table"
      >
        <thead
          class="table-headers"
        >
          <tr>
            <th
              class="table-header"
            >
              #
            </th>
            <th
              class="table-header"
            >
              header 1
            </th>
            <th
              class="table-header"
            >
              header 2
            </th>
          </tr>
        </thead>
        <tbody
          class="table-row"
        >
          <tr>
            <td
              class="table-cell"
            >
              0
            </td>
            <td
              class="table-cell"
            >
              Row 1 Content
            </td>
            <td
              class="table-cell"
            >
              Row 1 Content
            </td>
          </tr>
        </tbody>
        <tbody
          class="table-row"
        >
          <tr>
            <td
              class="table-cell"
            >
              1
            </td>
            <td
              class="table-cell"
            >
              Row 2 Content
            </td>
            <td
              class="table-cell"
            >
              Row 2 Content
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
